cmd/nodeup/main.go

```diff
@@ -64,17 +69,49 @@ func main() {
 	retries := flagRetries
 
 	for {
-		cmd := &nodeup.NodeUpCommand{
-			ConfigLocation: flagConf,
-			Target:         target,
-			CacheDir:       flagCacheDir,
-			FSRoot:         flagRootFS,
-			ModelDir:       models.NewAssetPath("nodeup"),
-		}
-		err := cmd.Run(os.Stdout)
-		if err == nil {
-			fmt.Printf("success")
-			os.Exit(0)
+		var err error
+		if install {
+			// create a systemd unit to bootstrap kops
+			// using the same args as we were called with
+			var command []string
+			for i := 0; i < len(os.Args); i++ {
+				s := os.Args[i]
+				if s == "-install" || s == "--install" {
+					continue
+				}
+				if i == 0 {
+					// We could also try to evaluate based on cwd
+					s, err = os.Readlink("/proc/self/exe")
```

Comment:
Should we

1) Check that file exists
2) Check that it is in fact a symlink

Before just bailing out here?